src/App.vue

```diff
@@ -14,17 +19,33 @@ import 'bootstrap-vue/dist/bootstrap-vue.css'
 import Navigation from './components/Navigation.vue'
 import Footer from './components/Footer.vue'
 import Homepage from './components/Homepage.vue'
+import HelloWorld from './components/HelloWorld.vue'
 import Vue from 'vue'
 // Install BootstrapVue
 Vue.use(BootstrapVue)
 // Optionally install the BootstrapVue icon components plugin
 Vue.use(IconsPlugin)
 export default {
   name: 'App',
+  data () {
+    return {
+      activeBody: 'home' // visible to template
```

Comment:
You can remove the comments bevor merging  
```suggestion
      activeBody: 'home'
```